src/CircuitBreaker/test/Hystrix.Test/InterruptibleCommand.cs

```diff
@@ -12,15 +12,14 @@ internal sealed class InterruptibleCommand : TestHystrixCommand<bool>
 
     public bool HasBeenInterrupted => _hasBeenInterrupted;
 
-    public InterruptibleCommand(TestCircuitBreaker circuitBreaker, bool shouldInterrupt, bool shouldInterruptOnCancel, int timeoutInMillis)
```

Comment:
Seems odd these weren't used, but I don't expect us to spend time implementing them now so removal does seem right the right thing to do